webpack/assets/javascripts/react_app/common/document.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Whether or not the page is focused (beeing used atm)
+ * @return {boolean}
+ */
+export const doesDocumentHasFocus = () =>
```

Comment:
can you add a test for it?

Reply:
I don't think it actually make sense to test the `document.hasFocus()` itself.

Does it make sense to add a test that `doesDocumentHasFocus` calls `document.hasFocus()` and return its results?